peachjam/views/generic_views.py

```diff
@@ -75,12 +75,14 @@ def get_context_data(self, **kwargs):
                 ).values_list("date__year", flat=True)
             )
         )
+        labels = {"Author": "Regional body"}
 
         context["facet_data"] = {
             "years": years,
             "authors": authors,
             "courts": courts,
             "alphabet": lowercase_alphabet(),
+            "labels": labels,
```

Comment:
How will Musango get hold of `labels` in javascript to use the labels in the search filter interface?